block-explorer/v3.yml

```diff
@@ -0,0 +1,643 @@
+              schema:
+                $ref: '#/components/schemas/NotFound'
+  /global-snapshot/{hash}:
+    get:
+      tags:
+        - Global snapshot
+      summary: Get global snapshot by hash
+      parameters:
+        - name: hash
+          in: path
+          description: Hash of global snapshot to return
+          required: true
+          schema:
+            type: string
+      responses:
+        200:
+          description: Global snapshot for given hash
+          content:
+            'application/json':
+              schema:
+                $ref: '#/components/schemas/Snapshot'
+
+        404:
+          description: Not found
+          content:
+            'application/json':
+              schema:
+                $ref: '#/components/schemas/NotFound'
+  /global-snapshot/{ordinal}:
+    get:
+      tags:
+        - Global snapshot
+      summary: Get global snapshot by ordinal
+      parameters:
+        - name: ordinal
+          in: path
+          description: Ordinal of global snapshot to return
+          required: true
+          schema:
+            type: integer
+            format: int64
+      responses:
+        200:
+          description: Global snapshot for given ordinal
+          content:
+            'application/json':
+              schema:
+                $ref: '#/components/schemas/Snapshot'
+        404:
+          description: Not found
+          content:
+            'application/json':
+              schema:
+                $ref: '#/components/schemas/NotFound'
+  /global-snapshot/latest/rewards:
+    get:
+      tags:
+        - Global snapshot
+      summary: Get rewards for latest global snapshot
+      responses:
+        200:
+          description: Rewards for latest global snapshot
+          content:
+            'application/json':
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/RewardTransaction'
+        404:
+          description: Not found
+          content:
+            'application/json':
+              schema:
+                $ref: '#/components/schemas/NotFound'
+  /global-snapshot/{hash}/rewards:
+    get:
+      tags:
+        - Global snapshot
+      summary: Get rewards for given global snapshot
+      parameters:
+        - name: hash
+          in: path
+          description: Hash of global snapshot to return rewards from
+          required: true
+          schema:
+            type: string
+      responses:
+        200:
+          description: Rewards for given global snapshot
+          content:
+            'application/json':
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/RewardTransaction'
+        404:
+          description: Not found
+          content:
+            'application/json':
+              schema:
+                $ref: '#/components/schemas/NotFound'
+  /global-snapshot/{ordinal}/rewards:
```

Comment:
Endpoint's name has `rewards` word in plural form, whereas all enpoints with `transaction` have singular form. I think on convention should be followed.